lib/resty/limit/req.lua

```diff
@@ -4,19 +4,19 @@
 local floor = math.floor
 local tonumber = tonumber
 
-
 local _M = { _VERSION = "0.01", OK = 1, BUSY = 2, FORBIDDEN = 3 }
 
-
-local redis_limit_req_script_sha
-local redis_limit_req_script = [==[
+local redis_limit_req_lookup_script_sha
+local redis_limit_req_lookup_script = [==[
```

Comment:
`Literal strings can also be defined using a long format enclosed by long brackets.`

Something like heredoc.